pkg/terraform/terraform.go

```diff
@@ -159,6 +171,41 @@ func hasBackendConfig(path string) (bool, hcl.Diagnostics, error) {
 	return false, diags, nil
 }
 
+// ParseBackendConfig extracts the backend configuration from the backend block.
+func ParseBackendConfig(path string) (*TerraformBackendConfig, hcl.Diagnostics, error) {
```

Comment:
Nit: `ExtractBackendConfig` might be a better name.

Reply:
Done